.github/workflows/unit-tests.yml

```diff
@@ -309,6 +315,8 @@ jobs:
           /etc/dcm.properties          
           /etc/authService.conf
           /version.txt
+          /lib/rdk/getMaintenanceStartTime.sh
```

Comment:
Did you created this new script for this work?  AS per RDK-E guideline we are removing the usage of script from cpp code.

Reply:
This is a mock script to read MM start time. These changes are giving permission to read and write file in docker. Maintenance manager UT will create this mock file and maintenance manager will read it. Whenever we change maintenance manager for RDK-E. The testcase needs to be updated for new design as per requirements.